src/components/pages/home/jumbotron/index.tsx

```diff
@@ -78,32 +71,23 @@ const Jumbotron: FunctionComponent<JumbotronProps> = ({
                   <h1>{title}</h1>
                 </a>
               </Link>
-              <Link href={instructor.path}>
-                <a
-                  className="mt-4 flex items-center space-x-2 text-base group"
-                  onClick={() =>
-                    track('clicked instructor in jumbotron', {
-                      instructor: instructor.slug,
-                    })
-                  }
-                >
```

Comment:
we don't link to instructors anywhere else on the page so removed that from the jumbotron